x-pack/legacy/plugins/siem/public/components/paginated_table/index.tsx

```diff
@@ -160,25 +159,13 @@ export const PaginatedTable = memo<SiemTables>(
       loadPage(newActivePage);
       updateActivePage(newActivePage);
     };
-    if (!isEmpty(pageOfItems) && isEmptyTable) {
-      setEmptyTable(false);
-    }
-    if (loading && isEmptyTable) {
-      return (
-        <EuiPanel>
-          <LoadingPanel
-            height="auto"
-            width="100%"
-            text={`${i18n.LOADING} ${loadingTitle ? loadingTitle : headerTitle}`}
-            data-test-subj="InitialLoadingPanelPaginatedTable"
-          />
-        </EuiPanel>
-      );
+    if (headerCount >= 0 && loadingInitial) {
```

Comment:
I think that should be inside of `useEffect`

Reply:
👍